mappings/net/minecraft/block/Block.mapping

```diff
@@ -433,7 +433,7 @@ CLASS bmm net/minecraft/block/Block
 	METHOD h hasBlockEntity ()Z
```

Comment:
Move all the mapping changes out of this PR please and into their own

Reply:
Removed.